spec/refresher_spec.rb

```diff
@@ -5,70 +5,70 @@
   context "guess the collection" do
 
     it "Yes, it is a Object, but which class" do
-      expect([].is_a? _fill_in_object_).to be true
-      expect(Array.new.instance_of? _FILL_ME_IN_).to be true
+      expect([].is_a? Array).to be true
+      expect(Array.new.instance_of? Array).to be true
     end
 
     it "Yes, it is a Object, but which class" do
-      expect({}.is_a? _fill_in_object_).to be true
-      expect(Hash.new.instance_of? _FILL_ME_IN_).to be true
+      expect({}.is_a? Hash).to be true
+      expect(Hash.new.instance_of? Hash).to be true
     end
   end
 
   context "guess the type" do
     it "Yes, it is a Object, but which" do
-      expect(:class.is_a? _fill_in_object_).to be true
+      expect(:class.is_a? Symbol).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect("module".is_a? _fill_in_object_).to be true
+      expect("module".is_a? String).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect(1.is_a? _fill_in_object_).to be true
+      expect(1.is_a? Integer).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect(1.5.is_a? _fill_in_object_).to be true
+      expect(1.5.is_a? Float).to be true
     end
   end
 
   context "the following are Array methods" do
     it "adds a thing to the end of the method" do
-      expect([]._fill_in_method_here_(1)).to eq [1]
+      expect([].Push(1)).to eq [1]
```

Comment:
@mikegee 
i fixed it...i was having trouble getting this pass...

 it "removes an item from the front of an array" do
      expect(["Banana", 1].delete_at(1)).to eq 1
    end

it works as it is, but I'm confused because when I thought an array index starts at 0, therefore by running "delete_at(1)" it would delete the integer 1, and leave banana.  However it does the opposite.

Thanks